modules/chmgr/chmgr.js

```diff
@@ -105,8 +106,10 @@ async function delete_unused_categories_interval() {
 }
 
 async function check_role(user, guild, cmd) {
-	if (user.bot == true)
+	if (user.bot) {
```

Comment:
in javascript this function if(user.bot === true) is not the same to this if(user.bot).

if(user.bot) is the same to if(user.bot !=== undefined)

Reply:
I assumed user.bot is either true or false in every case. if (user.bot) evaluates to true when user.bot is a number neq 0 or a non-empty string ..., but that shouldn't happen? If it's undefined or null, it always evaluates to false.

Reply:
You are right. when user.bot = true, the if(user.bot) statement is in fact true. So there is no problem with this line 👍